src/dsp_tools/commands/excel2json/resources.py

```diff
@@ -270,7 +272,7 @@ def _validate_excel_file(classes_df: pd.DataFrame, df_dict: dict[str, pd.DataFra
             f"Please use {[f'label_{lang}' for lang in languages]}"
         )
     problems: list[Problem] = []
-    if missing_super_rows := [int(index) + 2 for index, row in classes_df.iterrows() if not check_notna(row["super"])]:
+    if missing_super_rows := [int(str(i)) + 2 for i, row in classes_df.iterrows() if not check_notna(row["super"])]:
```

Comment:
I don't understand why it has to become a string first?

Reply:
Because a `Hashable` cannot be casted to an `int` directly. mypy wants to have the detour via str.

Reply:
Why does it work then?

Reply:
Ok, thanks for the explanation. I am not sure if the mypy logic makes sense to me, because not every string can be converted to an int, but that is besides the point.